best-practices/README.md

```diff
@@ -183,7 +183,9 @@ Email
 JavaScript
 ----------
 
-* Use CoffeeScript.
+* Use ES6 when possible with a tool like [6to5].
```

Comment:
We have an ongoing experiment related to ES6: https://trello.com/c/EGpNLWB1/393-es6-instead-of-coffeescript

It seems like the winds are blowing this way, but I think we should try out ES6 on a JavaScript-heavy project before determining it's ready to replace Coffeescript on projects going forward.

I think it would be fine to get guidelines in place for HOW to use ES6 in the event that you are using it. I just think we should have a JavaScript ES6 section instead of replacing the existing Coffeescript recommendation.

It could also be reasonable to soften this to something like, "Use Coffeescript, ES6 with 6to5, or another language that compiles to JavaScript." Basically, make the stance that JavaScript is a compile target.